src/main/java/org/pesmypetcare/webservice/controller/petmanager/PetRestController.java

```diff
@@ -150,6 +150,24 @@ public void deleteFieldCollection(@PathVariable String owner, @PathVariable Stri
         petService.deleteFieldCollection(owner, name, field);
     }
 
+    /**
+     * Deletes all the field collection elements with a key previous or smaller to the specified one.
+     * @param owner Username of the owner of the pet
+     * @param name Name of the pet
+     * @param field Name of the field where the action will be done
+     * @param key Specified key (This one not included)
+     * @throws DatabaseAccessException If an error occurs when accessing the database
+     * @throws DocumentException When the document does not exist
+     */
+    @DeleteMapping("/{owner}/{name}/fullcollection/{field}/{key}")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5506329)